emcc

```diff
@@ -1289,6 +1289,10 @@ try:
   # Emscripten
   logging.debug('LLVM => JS')
   extra_args = [] if not js_libraries else ['--libraries', ','.join(map(os.path.abspath, js_libraries))]
+  if memory_init_file:
+    shared.Settings.MEM_INIT_METHOD = 1
+  elif shared.Settings.MEM_INIT_METHOD == 1:
```

Comment:
Instead of silently doing this, let's assert that `MEM_INIT_METHOD` should not be 1 (if not `memory init file`)